knowledge/cars/delorean/qna.yaml

```diff
@@ -0,0 +1,19 @@
+created_by: zsplaingard
+domain: automobile
+seed_examples:
+- answer: | The first production DeLorean was completed on January 21, 1981.
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!